### DISCUSSION TITLE
Tutorials or sample model for `message-ix-models`?

### DISCUSSION THREAD

**Discussion by zsalimian** [iiasa/message_ix#531] [MESSAGE modeling] [answered]
Dear Sir,

Is it possible to send me a simple code for a simple RES in message-ix-models?

For instance, westeros example helped me a lot in learning MESSAGE-ix. Is there any similar code for message-ix-models?

**Comment by LauWien** [comment]
Dear @zsalimian,
Since this is neither about (a) a bug or error, nor (b) a requested feature in the software per se, I have transferred and converted this from an issue of the message-ix-models repository to a “Discussion” in the message_ix repository.

**Comment by khaeru** [comment, marked as answer]
As written by @LauWien on one of the (multiple) duplicates of this question (https://github.com/iiasa/message_ix/discussions/550#discussioncomment-2079272), there are currently no dedicated tutorials for advanced model-building using `message-ix-models`; only the detailed documentation at https://docs.messageix.org/models/.
